lib/active_merchant/billing/gateways/beanstream/beanstream_default.rb

```diff
@@ -0,0 +1,178 @@
+require 'active_merchant/billing/gateways/beanstream/beanstream_core'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class BeanstreamDefautGateway < Gateway
+      include BeanstreamCore
+
+      def initialize(options = {})
+        requires!(options, :login)
```

Comment:
let's leave this in BeanstreamCore to avoid modifying that file, since it could be used elsewhere

Reply:
Fixed.